UserSettings/EditorUserSettings.asset

```diff
@@ -6,31 +6,31 @@ EditorUserSettings:
   serializedVersion: 4
   m_ConfigSettings:
     RecentlyUsedScenePath-0:
-      value: 22424703114646680e0b0227036c6c111b07142f1f2b233e2867083debf42d
+      value: 224247031146466e0307183b370f6b365932002b21382a353e662e30ebc63d10e7eb3becf30e3a2f2d31ec390a3b4f2afc031d12
```

Comment:
Please reset changes to this file. It's not needed in this pull request

Reply:
Done